keyboards/keycapsss/kimiko/rev2/info.json

```
@@ -0,0 +1,192 @@
+{
+    "keyboard_name": "Kimiko Rev2",
+    "manufacturer": "Keycapsss",
+    "url": "https://keycapsss.com",
+    "maintainer": "BenRoe",
+    "usb": {
+        "vid": "0x7983",
+        "pid": "0x4B69",
+        "device_version": "2.0.0"
+    },
+    "diode_direction": "COL2ROW",
+    "debounce": 5,
+    "matrix_pins": {
+        "cols": ["F6", "F7", "B1", "B3", "B2", "B6"],
+        "rows": ["D4", "C6", "D7", "E6", "B4", "B5"]
+    },
+    "features": {
+        "mousekey": true,
+        "extrakey": true,
+        "encoder": true
+    },
```

migrated from `config.h`. rgblight not enabled as per conditional in `config.h`
```suggestion
    },
    "rgblight": {
        "split": true,
        "led_count": 62,
        "split_count": [31, 31]
    },
```